### PR TITLE
Make GEV tutorials to work

### PR DESCRIPTION
**Pull request by GianmarcoCallegher**

### CODE REVIEW

**Comment by GianmarcoCallegher** [comment]
@wiep would you review?

File: docs/source/tutorials/qmd/03-gev.qmd [hunk 1]
```diff
@@ -59,7 +59,7 @@ model <- liesel(
 )
 ```
 
-Now, we can continue in Python and use the `lsl.dist_reg_mcmc()` function to set up a sampling algorithm with IWLS kernels for the regression coefficients ($\boldsymbol{\beta}$) and a Gibbs kernel for the smoothing parameter ($\tau^2$) of the spline. Note that we need to set $\beta_0$ for $\xi$ to 0.1 manually, because $\xi = 0$ breaks the sampler.
+Now, we can continue in Python and use the `lsl.dist_reg_mcmc()` function to set up a sampling algorithm with IWLS kernels for the regression coefficients ($\boldsymbol{\beta}$) and a Gibbs kernel for the smoothing parameter ($\tau^2$) of the spline. Note that we need to set $\beta_0$ for $\xi$ to 0.1 manually, because $\xi = 0$ breaks the sampler. In order to avoid numerical instabilty and consequently invalid supports, we do not appy any jittering to $\boldsymbol{\beta}$ and $\tau^2$.
```
Comment (wiep) [inline docs/source/tutorials/qmd/03-gev.qmd:62]: The reader who doesn't know about the support of the GEV distribution might be confused. I think the support varying with parameters is not mentioned in the tutorial. I would write something like

> The support of the GEV distribution changes with the parameter values (compare [Wikipedia](https://en.wikipedia.org/wiki/Generalized_extreme_value_distribution)). To ensure that the initial parameters, support the observed data we set xi = 0 and disable jittering of the the variance and regression parameters. For the latter, we supply functions user-defined jitter functions to `lsl.dist_reg_mcmc` that are essentially the identity function wrt the parameter value.

File: liesel/model/distreg.py [hunk 2]
```diff
@@ -30,6 +31,13 @@
 
 matrix_rank = np.linalg.matrix_rank
 
+tau2_jitter_fn: JitterFunction = lambda key, val: jax.random.uniform(
+    key, val.shape, val.dtype, 1.0, 1000.0
+)
```
[the remark this answers is not in the document]
Comment (GianmarcoCallegher) [reply liesel/model/distreg.py:36]: I know it is not real jittering. We talked about it with Hannes and thought this was a simple way of ensuring that `tau2 > 0`. I'm fine with your method too.

File: docs/source/tutorials/qmd/03-gev.qmd (hunk 1 above)
Comment (GianmarcoCallegher) [reply docs/source/tutorials/qmd/03-gev.qmd:62]: Ok. Thanks.

File: liesel/model/distreg.py (hunk 2 above)
[the remark this answers is not in the document]
Comment (GianmarcoCallegher) [reply liesel/model/distreg.py:36]: We have always used something like 1000. I can reduce it

**Comment by GianmarcoCallegher** [comment]
Wanna give it a second look?

File: liesel/model/distreg.py (hunk 2 above)
[the remark this answers is not in the document]
Comment (wiep) [reply liesel/model/distreg.py]: keep it if it makes sense. I think you discussed that with @hriebl. I don't want to object.

**Comment by wiep** [comment]
I think it's ready to be merged but there's a merge conflict.

**Review by wiep — APPROVED** [review]

**Comment by wiep** [comment]
@GianmarcoCallegher, i think we still have a numerical instability that showed up when the server rendered the qmd files. One chain got stuck.

```
liesel.goose.engine - INFO - Starting epoch: POSTERIOR, 1000 transitions, 25 jitted together
liesel.goose.engine - WARNING - Errors per chain for kernel_00: 1, 1, 1000, 1 / 1000 transitions
```

**Comment by GianmarcoCallegher** [comment]
Weird. On my laptop, it works with no problem

**Pull request merged**